src/app/modules/main/communities/models/curated_community_item.nim

```diff
@@ -11,6 +11,7 @@ type
     color: string
     tags: string
     members: int
+    activeMembers: int
```

Comment:
just for myself. What's the difference between members and activeMembers? Do active members - members, who are online now?